tests/executors/test_celery_executor.py

```diff
@@ -557,7 +559,9 @@ def test_the_check_interval_to_clear_stuck_queued_task_is_correct_for_db_query(
             mock_backend = DatabaseBackend(app=celery_executor.app, url="sqlite3://")
             with mock.patch('airflow.executors.celery_executor.Celery.backend', mock_backend):
                 mock_session = mock_backend.ResultSession.return_value
-                mock_session.query.return_value.all.return_value = [result_obj("SUCCESS", task_id)]
+                mock_session.query.return_value.filter.return_value.all.return_value = [
```

Comment:
After mocking this query, because the `update_all_task_states` has the same query at BulkStateFetcher, it throws an error. I had to mock the `update_all_task_states` too